syntax/swift.vim

```diff
@@ -150,6 +150,7 @@ syntax keyword swiftKeywords
       \ required
       \ rethrows
       \ return
+      \ #selector
```

Comment:
Sure, I mean it makes sense since it's starts with a #, the swift evolution notes isn't too clear on what it actually is, except that it's an expression